Project-II-Calculator/calculator/src/components/DisplayComponents/Display.css

```diff
@@ -0,0 +1,72 @@
+.calculatorDisplay {
+  width: 360px;
+  height: 420px;
+  display: flex;
+  flex-direction: column;
+}
+
+.wholeCalc {
+  display: flex;
+}
+
+.calculatorAnswer {
+  width: 100%;
+  background: rgb(32, 27, 27);
+  color: white;
+  font-size: 5rem;
+  text-align: right;
+  border: 1px solid gray;
+  padding: 0 2% 0 0;
+}
+
+.calculatorLeft {
+  width: 75%;
+  display: block;
+}
+
+.calculatorRight {
+  width: 25%;
+  display: flex;
+  flex-direction: column;
+}
+
+.clear {
+  width: 100%;
+  height: 20%;
+  background: white;
+  border: 1px solid gray;
+  font-size: 2.6rem;
+  font-weight: lighter;
+}
+
+.numberBlock {
+  display: flex;
+  flex-direction: column-reverse;
```

Comment:
This is a good example of when to use flex-column, since you also need it's reverse functionality. However, it would be better practice to just organize the buttons in the order you want them to appear in. For someone using a screen reader to use your app, there is no indicator for them that the buttons are not in the order they hear them in until they press a button.